packages/neutrino/extensions.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   // Modifying a value here should have an accompanying change there as well.
   // We can't pull in neutrino there as that would potentially give us
   // conflicting versions in node_modules.
-  source: ['js', 'jsx', 'vue', 'ts', 'tsx', 'mjs'],
+  source: ['wasm', 'mjs', 'jsx', 'vue', 'tsx', 'ts', 'web.jsx', 'web.js', 'js'],
```

Comment:
Bah the rest of my comment got lost, since I forgot to press update comment before submitting the review. Trying again:

The value for `source` ends up in `neutrino.options.extensions`, which is used for `neutrino.config.resolve.extensions` as expected, but also used by the Mocha preset for the Babel extensions regex, and the Jest preset for `moduleFileExtensions`. Do the latter two uses support wasm, or does wasm need to just be for `neutrino.config.resolve.extensions` (ie webpack) only? (Similar to how `json` is added only for webpack by several of the presets: https://github.com/neutrinojs/neutrino/blob/c11b1c987e6a93491f3765b733e98bf66fe4fc61/packages/web/index.js#L166-L169)

Reply:
My guess is yes, so that those files can be imported from tests. Correct me if you think this assumption is invalid.